iOS/src/toga_iOS/app.py

```diff
@@ -105,7 +109,10 @@ def set_icon(self, icon):
         pass  # pragma: no cover
 
     def set_main_window(self, window):
-        pass
+        if window is None:
+            raise ValueError("Session-based apps are not supported on Android")
+        elif window == toga.App.BACKGROUND:
+            raise ValueError("Background apps are not supported on Android")
```

Comment:
If the term "session-based" is removed from the documentation, then both of these messages (in all backends) could be replaced by "Apps without a main window are not supported".

Reply:
Also, "Android" is a cut and paste error.